src/est/est.c

```diff
@@ -9,19 +9,38 @@
  *
  * November, 2012
  *
- * Copyright (c) 2012-2014 by cisco Systems, Inc.
+ * Copyright (c) 2012-2014, 2016 by cisco Systems, Inc.
```

Comment:
just an observation, but for me, the above diff is not highlighted correctly, or at least like this visual diff tool has done up to this point.  All previous occurrences it would highlight in dark green any additions, but here it does not highlight the change from '-' to ',' and the addition of 2016.  Not sure why.